docs/source/code-generation/codegen-configuration.mdx

```diff
@@ -667,3 +630,72 @@ let configuration = ApolloCodegenConfiguration(
 </MultiCodeBlock>
 
 For more details, see the section on [downloading a schema](./downloading-schema).
+
+## Operation Manifest Configuration
+
+Optional settings used to configure the usage of [Persisted Queries or Automatic Persisted Queries](./../fetching/persisted-queries) in order to generate the operation identfier manifest and determine how operation files are generated.
+
+| Property Name | Description |
+| ------------- | ----------- |
+| `path` | Local path where the generated operation manifest file should be written. |
+| [`version`](https://www.apollographql.com/docs/ios/docc/documentation/apollocodegenlib/apollocodegenconfiguration/operationmanifestfileoutput/version) | The version format to use when generating the operation manifest. |
+| [`generateManifestOnCodeGeneration`](https://www.apollographql.com/docs/ios/docc/documentation/apollocodegenlib/apollocodegenconfiguration/operationmanifestconfiguration/autogenerate) | Whether or not the operation manifest should be generated every time code generation is run. Defaults to false. |
+
+<MultiCodeBlock>
+
+```json title="CLI Configuration JSON"
+"operationManifestConfiguration" : {
+  "generateManifestOnCodeGeneration" : false,
+  "path" : "/operation/identifiers/path",
+  "version" : "persistedQueries"
+}
+```
+
+```swift title="Swift Codegen Setup"
+let configuration = ApolloCodegenConfiguration(
+    // Other properties not shown
+    operationManifestConfiguration: .init(
+        path: "./manifest/operationManifest.json",
+        version: .persistedQueries,
+        generateManifestOnCodeGeneration: false
+    )
+)
+```
+
+</MultiCodeBlock>
+
+### Operation Manifest
+
+Providing a value for this property will generate a JSON document with all your operations and their computed identifier hashes. This document can be used to pre-register the identifiers for your operations with a server that supports [persisted queries or Automatic Persisted Queries](./../fetching/apqs).
+
+The properties of the Operation Manifest configuration object are:
+
+| Property Name | Description |
+| ----- | ----------- |
+| `path` | Local path where the generated operation manifest file should be written. |
+| [`version`](https://www.apollographql.com/docs/ios/docc/documentation/apollocodegenlib/apollocodegenconfiguration/operationmanifestfileoutput/version) | The version format to use when generating the operation manifest. |
+
+<MultiCodeBlock>
+
+```json title="CLI Configuration JSON"
+"output": {
+    "operationManifest" : {
+        "path" : "./generated/operationIdentifiers.json",
+        "version" : "persistedQueries"
+    }
+}
+```
+
+```swift title="Swift Codegen Setup"
+let configuration = ApolloCodegenConfiguration(
+    // Other properties not shown
+    output: ApolloCodegenConfiguration.FileOutput(
+        operationManifest: .init(
+            path: "./generated/operationIdentifiers.json",
+            version: .persistedQueries
+        )
+    )
+)
+```
+
+</MultiCodeBlock>
```

Comment:
I think this section needs to be deleted? It's an older version of whats directly above it.